mava/systems/system.py

```diff
@@ -64,19 +64,19 @@ def _make_config(self) -> None:
                 input = {component.name(): config_class()}
                 self.config.add(**input)
 
-    @abc.abstractmethod
```

Comment:
Fixed.

Reply:
The abstract class is to ensure that every other class that inherits from the System class has to overwrite the design method, otherwise, it will complain. It just acts as a basic check in some sense and helps users understand what they need to overwrite etc., which is good practice. Not sure if you also tried adding it to the design method in `IPPOSystem` which inherits from `System`? Because that we shouldn't do, since in IPPO (and other systems) we are overwriting the method.

Reply:
That makes sense, thanks. I have learned something new today :) That seems to have been the issue. I have added the abstract method back in the BaseSystem.